ethereum_history_api/circuits/get_account/src/main.nr

```diff
@@ -0,0 +1,14 @@
+use dep::ethereum_history_api::account::{verify_account, Account};
+use dep::ethereum_history_api::header::{get_header, BlockHeaderPartial};
+use dep::ethereum_history_api::misc::types::{Address, Bytes32};
+
+fn main(
+    block_no: pub Field,
+    address: pub Address,
+    // This is marked pub due to an issue in noir JS library. For details - see README.md `Private arguments` section.
+    state_root: pub Bytes32
+) -> pub Account {
```

Comment:
Let's extract this method to getAccount in `lib/account` and make the main just call this method.

Reply:
done